### PR TITLE
Fix footer bugs

### PR DESCRIPTION
**Pull request by libertymayc**
#861

### CODE REVIEW

**Review by Fercas123 — APPROVED** [review]

File: site/src/css/custom.css [hunk 1]
```diff
@@ -339,7 +339,11 @@ ul.table-of-contents li > ul li {
   --ifm-footer-background-color: var(--uitk-color-gray-900);
   --ifm-footer-link-color: var(--uitk-text-secondary-foreground);
   --footer-items-separator-height: 16px;
-  --ifm-footer-padding-vertical: calc(var(--uitk-size-unit) * 1);
+  --ifm-footer-padding-vertical: calc(var(--uitk-size-unit) * 1.5);
```
Comment (JoanaMMoreira) [inline site/src/css/custom.css:342]: Can we just check with @shey-v if she's happy with the footer height please?

File: site/src/css/custom.css [hunk 2]
```diff
@@ -359,14 +363,15 @@ ul.table-of-contents li > ul li {
   max-width: none;
 }
 
+
```
Comment (james-nash) [inline site/src/css/custom.css:366]: This extra blank line is what's causing the linting check to fail on CI.

(You can configure VSCode to use Prettier to auto-format your files whenever you save them. That can avoid these kinds of issues)

**Review by james-nash — CHANGES_REQUESTED** [review]
Mostly fine, just need to fix a minor linting issue

**Review by shey-v — CHANGES_REQUESTED** [review]
For the max-width: 996px breakpoint, the left and right padding is lost. We need to add this in as per the other breakpoints (16px). 

Overall, the padding still looks quite tight, let's have a call in the morning. Thanks!

**Review by Fercas123 — APPROVED** [review]

**Review by shey-v — APPROVED** [review]
Looks good. Thanks!

**Review by james-nash — APPROVED** [review]
I hereby give my blessing to this PR 👼

**Pull request merged**